pkg/plugins/scms/github/pullrequest.go

```diff
@@ -199,20 +201,26 @@ func (p *PullRequest) CreatePullRequest(title, changelog, pipelineReport string)
 func (p *PullRequest) generatePullRequestBody() (string, error) {
 	t := template.Must(template.New("pullRequest").Parse(PULLREQUESTBODY))
 
+	fileUrl := p.gh.Spec.URL + p.gh.Spec.Owner + "/" + p.gh.Spec.Repository + "/tree/" + p.gh.HeadBranch + "/" + p.Filename
```

Comment:
I think it would be better to use the url.JoinPath like https://github.com/updatecli/updatecli/blob/c65cec948e3e90a479811979970e29b99d6c6d05/pkg/plugins/scms/github/main.go#L104

Normally your link should be working for Github Enterprise too

Reply:
While I really like the motivation, this approach won't catch PR opened from fork.
I think we have a missing part like just after reading a manifest, we would have a function that retrieve the local git information